packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,265 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { Description } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import { FormElementStory } from "./FormElement.stories.styles";
+import CheckboxExample from "./examples/CheckboxExample";
+import RadioExample from "./examples/RadioExample";
+import DatePickerExample from "./examples/DatePickerExample";
+import AccessibilityExample from "./examples/AccessibilityExample";
+import ListBoxExample from "./examples/ListBoxExample";
+import ButtonGroupExample from "./examples/ButtonGroupExample";
+import NativeInputExample from "./examples/NativeInputExample";
+import NestedExample from "./examples/NestedExample";
+import HTMLExample from "./examples/HTMLExample";
+import ComponentLabelExample from "./examples/ComponentLabelExample";
+import FormElement from "../src/FormElement";
+
+<Meta
+  title={`${getStoryName("FormElement")}/Docs`}
+  component={FormElement}
+  parameters={{ options: { showPanel: false } }}
+/>
+
+<Description markdown={Readme} />
+
+## More Examples
+
+### Index
+
+- <a href="#accessibility">Accessibility</a>
+- <a href="#alternate-layouts">Alternate Layout</a>
+
+  - <a href="#checkboxes">Checkboxes</a>
+  - <a href="#radios">Radios</a>
+  - <a href="#date-picker">Date Picker</a>
+  - <a href="#listbox">ListBox</a>
+  - <a href="#buttongroup">ButtonGroup</a>
+  - <a href="#nested-formelement">Nested FormElements</a>
+
+### Accessibility
+
+The Formelement has been designed with accessibiklity in mind. By using `<FormElement.Instructions>` users can add instructions for the end user. Users can also make use of the `<FormElement.Description>` to add additional direction for consumers.
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Instructions>
+    <span>
+      Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example text for
+      extra panel for questionnaires. Example text for extra panel for questionnaires
+    </span>
+  </FormElement.Instructions>
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <Input
+        id={idForLabel}
+        onChange={handleChange}
+        value={value}
+        placeholder="Form placeholder"
+        aria-describedby={ariaDescribedBy}
+        hasError={Boolean(errorText.length)}
+      />
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>This is description text</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <AccessibilityExample />
+</FormElementStory>
+
+### Alternate Layouts
```

Comment:
I don't think this is the right heading... actually, I'm not sure we need to nest the following use cases under a heading.  I know they were all embedded on an Alternate Layouts story before, but I much prefer the way you've broken them out individually.  Let's drop this heading and elevate Checkboxes, Radios, etc to `###` level.